src/dataServices/dynamoDbParamBuilder.ts

```diff
@@ -166,28 +175,28 @@ export default class DynamoDbParamBuilder {
         return params;
     }
 
-    static buildUpdateExportRequestJobStatus(jobId: string, jobStatus: ExportJobStatus) {
+    static buildUpdateExportRequestJobStatus(jobId: string, jobStatus: ExportJobStatus, tenantId?: string) {
         const params = {
             TableName: EXPORT_REQUEST_TABLE,
             Key: DynamoDBConverter.marshall({
-                jobId,
+                jobId: buildHashKey(jobId, tenantId),

```

Comment:
Could pull out `buildHashKey(jobId, tenantId)` call into a variable and use it in both places:
```ts
const jobId = buildHashKey(jobId, tenantId)
```